static/app/components/feedback/list/useListItemCheckboxState.tsx

```diff
@@ -0,0 +1,19 @@
+import {useCallback, useMemo, useState} from 'react';
+
+export default function useListItemCheckboxState() {
+  const [state, setState] = useState<Record<string, boolean>>({});
+
+  const checked = useMemo(() => Object.keys(state).filter(key => state[key]), [state]);
```

Comment:
nit/l: this is just Object.values

```suggestion
  const checked = useMemo(() => Object.values(state).filter(value => value), [state]);
```

Reply:
turns out it's not. I went and changed it, and TS helped me out here :D

We're returning the keys (strings, id's of each feedback) and filtering by whether they are checked or not. 

I'll rewrite it to separate the steps and be clearer.